packages/twenty-front/src/modules/object-record/object-options-dropdown/components/ObjectOptionsDropdownRecordGroupsContent.tsx

```diff
@@ -131,6 +145,7 @@ export const ObjectOptionsDropdownRecordGroupsContent = () => {
           </DropdownMenuItemsContainer>
         </>
       )}
+      {RecordGroupReorderConfirmationModal}
```

Comment:
We should be calling <RecordGroupReorderConfirmationModal />

Reply:
It's because `RecordGroupReorderConfirmationModal` can be null, so we can't directly call `< RecordGroupReorderConfirmationModal />`, I can do a check before but it's more common to use it that way in that particular case I guess